src/mds/topology/topology_service.cpp

```diff
@@ -127,6 +127,39 @@ void TopologyServiceImpl::GetChunkServer(
     }
 }
 
+void TopologyServiceImpl::GetChunkServerInCluster(
+    google::protobuf::RpcController* cntl_base,
+    const GetChunkServerInClusterRequest* request,
+    GetChunkServerInClusterResponse* response,
+    google::protobuf::Closure* done) {
+    brpc::ClosureGuard done_guard(done);
+
+    brpc::Controller* cntl =
+        static_cast<brpc::Controller*>(cntl_base);
+
+    LOG(INFO) << "Received request[log_id=" << cntl->log_id()
```

Comment:
Will this function be called frequently? If so, these logs may be a bit too much.

Reply:
fixed